sql_user_check_mysql/README.md

```diff
@@ -0,0 +1,19 @@
+# MYSQL User Check
+
+database_stats.sh script can be used to test connectivity with MYSQL database as well as fetch some stats and metadata. 
+The script supports 2 modes which can be specified with -m flag while running the script. One mode is stats mode which 
+will just print info like list of databases with their size. Count of tables, rows, column, size per schema. 
+Distribution of data by data type. Stats mode is the default mode. Another mode is full_metadata which will print 
+complete information of all columns like their data type, constraints etc.
```

Comment:
Mention in the readme somewhere that we will 3306 as default port if not provided as arg

Reply:
done